code/modules/surgery/tools.dm

```diff
@@ -86,6 +86,46 @@
 	origin_tech = "materials=1;biotech=1"
 	attack_verb = list("attacked", "slashed", "sawed", "cut")
 
+/obj/item/weapon/circular_saw/energy //For use by medical syndiborgs

```

Comment:
Can do. Any idea how to make it function as a circular saw for surgery purposes?